Assets/VRTK/Scripts/VRTK_AdaptiveQuality.cs

```diff
@@ -0,0 +1,708 @@
+// Adaptive Quality|Scripts|0240
+
+// Adapted from The Lab Renderer's ValveCamera.cs, available at
+// https://github.com/ValveSoftware/the_lab_renderer/blob/master/Assets/TheLabRenderer/Scripts/ValveCamera.cs
+// For The Lab Renderer's license see THIRD_PARTY_NOTICES.md
+namespace VRTK
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.Text;
+    using UnityEngine;
+    using UnityEngine.VR;
+
+    /// <summary>
+    /// Adaptive Quality dynamically changes rendering settings to maintain VR framerate while maximizing GPU utilization.
+    /// </summary>
+    /// <remarks>
+    /// The Adaptive Quality script is attached to the `eye` object within the `[CameraRig]` prefab.
+    /// <para>&#160;</para>
```

Comment:
I think things like this will just get ignored by the doc generator :(

same with the list, best to just use Markdown to do a list.

I should probably support things like this, but time ran out :(